packages/dataviews/src/filters.js

```diff
@@ -53,11 +39,11 @@ const Filters = memo( function Filters( { fields, view, onChangeView } ) {
 					isVisible:
 						isPrimary ||
 						view.filters.some(
-							( f ) =>
-								f.field === field.id &&
-								[ OPERATOR_IN, OPERATOR_NOT_IN ].includes(
-									f.operator
-								)
+							( f ) => f.field === field.id
+							// &&
```

Comment:
@oandregal can you help me understand the commented check? Is it needed?

Reply:
It checks the operator in use by the view is one of the available/registered operators. It essentially prevents filters with no operators (or with unknown operators) from being displayed. Operators are a required field in the `view.filters`, this check enforces that requirement.

Imagine the following situation:

- We store the filter in the view without operator, assuming the default in the UI is `IS`.
- For whatever reason, the UI changes the default to `IS NOT`. At this point, all the filters that come from the view without operator would have their results inverted (custom views from database, default views provided by WordPress, etc.).

Reply:
In summary, we still need it :)